test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/Import/SqlServerIndexesRebuildTests.cs

```diff
@@ -0,0 +1,448 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Features.Definition;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.Core.UnitTests.Extensions;
+using Microsoft.Health.Fhir.Shared.Tests.Integration.Features.Operations.Import;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema;
+using Microsoft.Health.Fhir.SqlServer.Features.Storage;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Integration.Persistence;
+using Microsoft.Health.JobManagement;
+using Microsoft.Health.SqlServer;
+using Microsoft.Health.SqlServer.Configs;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Schema.Manager;
+using Microsoft.Health.SqlServer.Features.Storage;
+using Microsoft.Health.Test.Utilities;
+using Microsoft.SqlServer.Dac.Compare;
+using NSubstitute;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Tests.Integration.Features.Operations.Import
+{
+    [Trait(Traits.OwningTeam, OwningTeam.FhirImport)]
+    [Trait(Traits.Category, Categories.Import)]
+    public class SqlServerIndexesRebuildTests : IClassFixture<SqlServerFhirStorageTestsFixture>
+    {
+        private const string LocalConnectionString = "server=(local);Integrated Security=true;TrustServerCertificate=True";
+        private const string MasterDatabaseName = "master";
+
+        public SqlServerIndexesRebuildTests()
+        {
+        }
+
+        [Fact]
+        public async Task GivenTwoDatabases_WhenOneDisableAndRebuildIndexes_ThenTwoDatabasesShouldBeTheSame()
```

Comment:
Could more of these two tests be factored out into common methods? Right now they are almost exact duplicates.

Reply:
Sorry, I will fix this problem.